hono-jsx/deps.ts

```diff
@@ -1 +1,2 @@
-export { jsx } from "https://deno.land/x/hono@v3.10.0/jsx/jsx-runtime.ts";
+export { jsx } from "npm:hono@3.12.9/jsx/jsx-runtime";
```

Comment:
Since Hono provides deno builds, I don't think importing from npm would be a good idea.

Reply:
This will download the entire `hono` npm package (most of which is not needed by the hono jsx plugin), thus increasing the dependency size.

Reply:
Ah, yeah that's better. Thanks for the feedback! :)